public/redux/reducers/anomalies.ts

```diff
@@ -24,12 +24,12 @@ import { Anomalies } from '../../models/interfaces';
 
 const PREVIEW_DETECTOR = 'ad/PREVIEW_DETECTOR';
 
-export interface Anomalies {
+export interface PreviewAnomalies {
```

Comment:
Minor: may want to change the filename completely to `previewAnomalies.ts` to keep it consistent

Reply:
will try to change it.